documentation/src/main/asciidoc/topics/proc_server_configuring_resp.adoc

```diff
@@ -49,18 +49,45 @@ The RESP endpoint automatically configures and starts a `respCache` cache. This
 * `application/octet-stream` encoding for both keys and values.
 * `RESPHashFunctionPartitioner` hash partitioner, which supports the CRC16 hashing used by Redis clients
 
-The cache configuration cannot enable capabilities that violate the RESP protocol.
-For example, specifying an incompatible hash partitioning function or a key encoding different from `application/octet-stream`.
+[discrete]
+=== Explicit configuration for cache
+
+User can explicitly specify the cache characteristics if the defaults doesn't fit, in case the cache configuration try to enable capabilities that violate the RESP protocol the server will raise
```

Comment:
It is possible to supply a custom configuration for the cache, as long as it does not violate the requirements of the RESP connector, in which case the server will raise an exception and will not start.